job-descriptions/software-engineer-backend.md

```diff
@@ -0,0 +1,66 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Software Engineer - Backend
+
+### About us
+
+We're building the new standard developer platform. Top tech companies have invested \$100Ms to build internal developer platforms for code search, code review, alerts, and automation. Sourcegraph provides this standard developer platform to every company, helping startups and large enterprises ship better software faster.
+
+[Our mission](https://sourcegraph.com/plan) is to dramatically increase the number of people who can understand and write code. By making code more accessible, we will democratize software development and accelerate innovations that bring the future sooner in transportation, health care, energy, AI, communication, space travel, etc.
```

Comment:
minor: The personality of most backend engineers I know generally zone out with sentences like this. Personally I would enjoy a job advert which starts off with the main reason I like working at Sourcegraph: writing systems software, automating the tools you interact with, custom databases / etc, smart and easy-going co-workers. So I would like something closer to "About the role" part first, then later in the document we can put in the Silicon Valley speak.

Reply:
Acknowledged. I will think about reworking this section (especially now that we have the handbook), but don't want to bundle it in this PR. This section is standard across all our job descriptions so I want to think about whether we want to diverge from the standard or update the standard format.